### PR TITLE
Cumulus 1265 lambdas in vpc

### PR DESCRIPTION
**Pull request by laurenfrederick**
**Summary:** Put all lambdas in the VPC

Addresses [CUMULUS-1265](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1265)

## Changes

* Put all of the lambdas inside the vpc
* Update tests to remove `NoVpc` lambdas

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: docs/data-cookbooks/browse-generation.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ This will cause kes to export the workflows in the new file along with the other
 Ensure the following lambdas are in your deployment's lambdas.yml (reference the [example lambdas.yml](https://github.com/nasa/cumulus/blob/master/example/lambdas.yml)):
 
 ```
-DiscoverGranulesNoVpc:
+DiscoverGranules:
```
Comment (ifestus) [inline docs/data-cookbooks/browse-generation.md]: should this object and the following object in the `SyncGranule` change have `launchInVpc: true` set?

File: CHANGELOG.md [hunk 2]
```diff
@@ -58,8 +58,9 @@ As a result of the changes for **CUMULUS-1193** and **CUMULUS-1264**, **you must
 
 ## Added
 
-- **CUMULUS-1263**
+- **CUMULUS-1265**
   - Added `apiConfigs` configuration option to configure API Gateway to be private
+  - All internal lambdas configured to run inside the VPC by default
```
Comment (mennovandiermen) [inline CHANGELOG.md]:
```suggestion
  - All internal lambdas configured to run inside the VPC by default
  - Removed references to `NoVpc` lambdas from documentation and `example` folder.
```

File: docs/data-cookbooks/browse-generation.md (hunk 1 above)
Comment (laurenfrederick) [reply docs/data-cookbooks/browse-generation.md]: yeah I think you're probably right here.

Comment (laurenfrederick) [reply docs/data-cookbooks/browse-generation.md]: I updated it, thanks!

**Pull request merged**